third_party/ot.hs/.ghci

```diff
@@ -0,0 +1,4 @@
+:set -fwarn-unused-binds -fwarn-unused-imports
```

Comment:
I cannot review your changes to the ot library, because I have no way to tell what's yours, and what is part of the original.  And, sorry, I can't do a complete code review for the third party ot library.

Is it possible to see a diff for your changes somehow?  Have you submitted your changes upstream, so that CodeWorld isn't maintaining a fork of this library indefinitely?

Reply:
Sorry for this. There were no changes made by me over here. I will remove this and add a dynamic installation of ot.hs.

Reply:
Wait as a matter of fact I did made changes to the cabal file. The dependencies specified were not being resolved.

```
Resolving dependencies...
cabal: Could not resolve dependencies:
trying: ot-0.2.1.0 (user goal)
next goal: ghc (dependency of ot-0.2.1.0)
rejecting: ghc-8.0.1/installed-8.0... (conflict: ghc =>
binary==0.8.3.0/installed-0.8..., ot => binary>=0.5.1.1 && <0.8)
Dependency tree exhaustively searched.
```
Resolving this still gave more dependency errors. So I removed the versions. The only change is the following:

```
25,32c25,32
<   Build-depends:       base,
<                        text,
<                        aeson,
<                        attoparsec,
<                        QuickCheck,
<                        binary,
<                        either,
<                        mtl,
---
>   Build-depends:       base >= 4 && < 5,
>                        text >= 1.0 && < 1.3,
>                        aeson >= 0.7 && < 0.11,
>                        attoparsec >= 0.10.1.1 && < 1,
>                        QuickCheck >= 2.7 && < 2.9,
>                        binary >= 0.5.1.1 && < 0.9,
>                        either >= 4.1.2 && < 5,
>                        mtl >= 2.1.3.1 && < 3,
```